apps/asap-server/src/routes/user.route.ts

```diff
@@ -166,7 +166,22 @@ export const userRouteFactory = (
       throw Boom.forbidden();
     }
 
-    const result = await userController.update(userId, update);
+    const { onboarded, ...userProfileUpdate } = update;
+
+    const result = await userController.update(userId, userProfileUpdate);
+
+    if (typeof onboarded === 'boolean') {
```

Comment:
This is kind of weird. You could send false and it would reject.

Reply:
i dont think i understand?